theorems/T000083.md

```diff
@@ -12,6 +12,6 @@ refs:
 ---
 
 By non-discreteness choose a non-isolated point of \(X\).
-It has an arc connected neighborhood with two distinct points,
+It has an arc connected neighborhood with at least two distinct points,
 so by {T75} this neighborhood is uncountable.
```

Comment:
I imagine "{T75}" was meant to show as a link on the web page?  Somehow it does not display properly for me.  Should we change something?

Reply:
That feature isn't supported yet, but I'm using it in the meantime anyway.